documentation/homework/week4/src/components/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react'
+
+import { inject, observer } from 'mobx-react'
+
+import AssignItem from './AssignItem'
+import ViewRender from './ViewRender'
+import EditRender from './EditRender'
+import { locals } from '../utils'
+
+@inject('InitialStore')
+@observer
+export default class extends Component {
+  componentDidUpdate = (nextProps) => locals.save = [...nextProps.InitialStore.items]
```

Comment:
This works, but I'd rather see you use MobX's [autorun](https://mobx.js.org/refguide/autorun.html) function to achieve the same thing.